README.md

```diff
@@ -52,37 +52,55 @@ Then, import `SwiftSyntax` in your Swift code.
 ## Contributing
 
 ### Building SwiftSyntax from `master`
+
 Since SwiftSyntax relies on definitions in the main Swift repository to generate the layout of the syntax tree using `gyb`, a checkout of [apple/swift](https://github.com/apple/swift) is still required to build `master` of SwiftSyntax.
 
-To build the `master` version of SwiftSyntax, check `swift-syntax` and  `swift` out side by side
+To build the `master` version of SwiftSyntax, follow the following instructions:
+
+1. Check `swift-syntax` and  `swift` out side by side:
+
 ```
 - (enclosing directory)
   - swift
   - swift-syntax
 ```
 
-And run `swift-syntax/build-script.py`. SwiftSyntax is now being built with the Swift compiler installed on the system.
+2. Make sure you have a recent master Swift toolchain installed.
```

Comment:
could you also add a download link here for master toolchain? like `https://swift.org/download/#snapshots`